packages/components/src/components/horizontal-swipe/horizontal-swipe.tsx

```diff
@@ -72,7 +83,8 @@ const HorizontalSwipe = ({
                     style={{
                         transform:
                             (should_show_left_hidden_component && `translate(${left_hidden_component_width})`) ||
-                            (should_show_right_hidden_component && `translate(-${right_hidden_component_width})`),
+                            (should_show_right_hidden_component && `translate(-${right_hidden_component_width})`) ||
+                            'none', // I don't know what the default value should be in case of the above two conditions fail
```

Comment:
```js
style={{
    ...(should_show_left_hidden_component && {
        transform: `translate(${left_hidden_component_width})`,
    }),
    ...(should_show_right_hidden_component && {
        transform: `translate(-${right_hidden_component_width})`,
    }),
}}
```
@iman-fs Can do like this, Although would be better to extract it into a variable.

Reply:
@iman-fs I bet your code is self-descriptive here, no need to add this comment. what do you think? 🙂